pkg/controller/seed-controller-manager/addoninstaller/addoninstaller_controller.go

```diff
@@ -264,7 +264,7 @@ func (r *Reconciler) ensureAddons(ctx context.Context, log *zap.SugaredLogger, c
 	return nil
 }
 
-func (r *Reconciler) addonReconciler(ctx context.Context, cluster *kubermaticv1.Cluster, addon kubermaticv1.Addon) reconciling.NamedAddonReconcilerFactory {
+func (r *Reconciler) addonReconciler(_ context.Context, cluster *kubermaticv1.Cluster, addon kubermaticv1.Addon) reconciling.NamedAddonReconcilerFactory {
```

Comment:
Is this an interface function? if not, I think we can remove the context altogether..

Reply:
Removed in bc53dec1874ca38518c821bd18486ffb1902b634